Doc/library/tarfile.rst

```diff
@@ -557,7 +557,7 @@ be finalized; only the internally used file object will be closed. See the
    and fall back to the :func:`fully_trusted <fully_trusted_filter>` filter,
    whose dangerous behavior matches previous versions of Python.
 
-   In Python 3.14+, leaving ``extraction_filter=None`` will cause
+   In Python 3.14, leaving ``extraction_filter=None`` will cause
```

Comment:
Isn't that a bit redundant?

```suggestion
   Leaving ``extraction_filter=None`` will cause
```

And can't we merge this with the previous paragraph?